elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/RedisAggregationDataStoreTestHarness.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.framework;
+
+import com.yahoo.elide.core.datastore.DataStore;
+import com.yahoo.elide.datastores.aggregation.AggregationDataStore;
+import com.yahoo.elide.datastores.aggregation.cache.RedisCache;
+import com.yahoo.elide.datastores.aggregation.metadata.MetaDataStore;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.ConnectionDetails;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.SQLDialectFactory;
+import com.yahoo.elide.datastores.multiplex.MultiplexManager;
+import com.yahoo.elide.modelconfig.validator.DynamicConfigValidator;
+
+import redis.clients.jedis.JedisPooled;
+
+import java.util.Collections;
+import java.util.Map;
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+
+public class RedisAggregationDataStoreTestHarness extends AggregationDataStoreTestHarness {
```

Comment:
I have mixed feelings about duplicating the IT tests here.   In one sense, it is good that we are testing many pathways through the cache.  On the other hand, this PR is making modifications to a lot of unrelated code. 
 Is there a way to test the RedisCache without pulling in all the IT tests?  Also, it would be great if RedisCache could "just work" on models without requiring changes to them.

Reply:
I have separated the IT tests for Redis and Regular ones with Redis IT running a few of the test cases